xml/System/DateTime.xml

```diff
@@ -46,7 +46,17 @@
     <summary>Represents an instant in time, typically expressed as a date and time of day.</summary>
     <remarks>
       <format type="text/markdown"><![CDATA[  
+## Examples  
+ The following example demonstrates how to compare roughly equivalent <xref:System.DateTime> values, accepting a small margin of difference when declaring them equal.  
   
+# [C++](#tab/cpp)
```

Comment:
why are we using tabs for the sample code? they can already select the language on the language selector, no?